src/doc/dev_manual/SiteReliabilityEngineering.rst

```diff
@@ -251,13 +251,18 @@ Active SRE Issues Repo
 `issues-only repository <https://github.com/visit-dav/live-customer-response/issues>`_
 within the `visit-dav GitHub organization <https://github.com/visit-dav>`_. 
 For each new inquiry, the primary will file an issue ticket and assign themselves.
-When the inquiry is :ref:`resolved <sre_response_vs_resolution>`, the associated
-issue is closed. The primary will endeavor to capture all relevant information and
+
+The primary will endeavor to capture all relevant information and
 communications in this issue. The use of GitHub issues for this purpose has a number
 of advantages over other options such as email including better search/browse as well
 as support for attachments. For this reason, a number of steps were taken to integrate
 the ``visit-users@elist.ornl.gov`` email list with this issues-only repository.
 
+The SRE issue shall be closed when the associated inquiry is
+:ref:`resolved <sre_response_vs_resolution>`. Or, it will be closed and labeled
+``wontfix`` when 21 days pass in which when the user ceases to engage in any ongoing
```

Comment:
This sentence doesn't quite read right. Perhaps changing the end of the sentence to "after 21 days pass since the user last engaged in any conversation with VisIt developers to reach a resolution."